pyfar/dsp/normalize.py

```diff
@@ -0,0 +1,151 @@
+import numpy as np
+
+def normalize(input_signal, norm_type = 'time', operation = 'max', channelwise = 'max',
+              value = 'unnasigned', f = np.arange(0,22100),  fs = 44100):
+    """
+    Parameters
+    ----------
+    input_signal:
+        signal of the signal class
+        
+    norm_type:
+        'time' - normalizes time signal to 'value' 
+        'abs' - normalizes magnitude spectrum to 'value'
+        'dB' - normalizes db magnitude spectrum to 'value'
+        (default = 'time')
+
+    operation:
+        'max' - finds absolute maximum of data to normalize 
+        'mean' - find the mean of data to normalize
+        'rms' - fins the rms of data to normalize
+        (default = 'max')
+    
+    channelwise:
+        'each' - normalize each channel seperately 
+        'max' - normalize to max or operation across all channels
+        'min' - normalize to min or operation across all channels
+        'mean' - normalize to mean of operation across all channels
+        (default = 'max')
+    
+    value: 
+        normalizes to 'value' which can be a scalor or a vector 
+        with a number of elements equal to channels. Default is 0 
+        for type = 'db' and 1 otherwise
+    f:
+        two element vector specifiying upper and lower frequenzy bounds
+        for normalization or scalar specifying the centre frequency for 
+        normalization (see 'frac', default is [0,fs/2])
+    fs:
+        sampling frequency in Hz (default = 44100)
+        
+    Returns
+    --------
+    signal_normalized - normalized input signal
+    """
+    
+    # set defaults
+    
+    if value == 'unnasigned':
+        if norm_type == 'dB':
+            value = 0
+        else:
+            value = 1
```

Comment:
This can be shorter in Python :)
```suggestion
        value = 0 if norm_type == 'dB' else 1
```